### PR TITLE
Files for localisation of RDMO into other languages

### PR DESCRIPTION
**Pull request by Zack-83**
Contains a R script and TSV translation memories

### CODE REVIEW

**Review by dario-pilori — APPROVED** [review]
I corrected some typos, but everything else looks fine to me. I guess that, in Italian, "Personenmonaten" is more often translated into "mesi-uomo" instead of "persone-mesi", but grammatically they're both correct.

File: localisation/rdmo-options-localisation.tsv [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+/rdmo/option[57]/text	CLARIN	CLARIN	CLARIN	CLARIN
+/rdmo/option[58]/text	Core TrustSeal	Core TrustSeal	Core TrustSeal	Core TrustSeal
+/rdmo/option[59]/text	DINI	DINI	DINI	DINI
+/rdmo/option[60]/text	TRAC	TRAC	TRAC	TRAC
+/rdmo/option[61]/text	DIN 31644	DIN 31644	DIN 31644	DIN 31644
+/rdmo/option[62]/text	Weiß es nicht	Don't know	Pas connu	Sconosciuto
+/rdmo/option[63]/text	Sonstiges	Other	Autre	Altro
+/rdmo/option[64]/text	Keine Zertifizierung	No certification	Aucune certification	Nessuna certificazione
+/rdmo/option[65]/text	Mehrfache Kopien der Daten	Multiple copies of the files	Copies multiples des fichiers	Copie multiple dei file
+/rdmo/option[66]/text	Verschlüsselung der Daten	Cryptography of the data	cryptage des données	criptaggio dei dati
```
[the remark this answers is not in the document]
Comment (Zack-83) [reply localisation/rdmo-options-localisation.tsv]: avrei preferito una parola descrivente l'azione ma non la trovo (Esiste "crittazione?")

File: localisation/rdmo-translation.R [hunk 2]
```diff
@@ -0,0 +1,52 @@
+library(xml2); library(stringr); library(magrittr); library(tidyverse)
+setwd("C:/Users/lanza01/Desktop/lavori in corso/RDMO 2020")
```
Comment (jmfrenzel) [inline localisation/rdmo-translation.R:2]: This seems to be a local path. Do we need this here?

**Review by jmfrenzel — CHANGES_REQUESTED** [review]
The tool is of great need. 

I wonder if you can add an README.md file with a short documentation how this tool should be used.
Is there a workflow automatize translations using this tool.

See also inline comments.

File: localisation/rdmo-translation.R (hunk 2 above)
Comment (Zack-83) [inline localisation/rdmo-translation.R:2]:
```suggestion
# setwd("C:/Users/MYPATH")    # enter path
```

**Comment by Zack-83** [comment]
> 
> 
> The tool is of great need.
> 
> I wonder if you can add an README.md file with a short documentation how this tool should be used.
> Is there a workflow automatize translations using this tool.
> 
> See also inline comments.

I added the README.md . The workflow is not automatic yet and requires several manual steps by the user. There is still much room for improvement.

**Review by jmfrenzel — CHANGES_REQUESTED** [review]
The changes seem ok to me. 

However following the instructions  `localisation/README.md` I  understand that  `*.tsv` files are being created from "source" xml. As a consequencewe do not need `tsv` files in version control. I suggest to remove them from the PR

**Comment by Zack-83** [comment]
> 
> 
> The changes seem ok to me.
> 
> However following the instructions `localisation/README.md` I understand that `*.tsv` files are being created from "source" xml. As a consequencewe do not need `tsv` files in version control. I suggest to remove them from the PR

We can remove them at this stage. In the future we might need them again if we have to communicate with translators who cannot run scripts.
Hey @dario-pilori, how can I remove a file from GitHub?

**Comment by dario-pilori** [comment]
> Hey @dario-pilori, how can I remove a file from GitHub?
I just removed it using command-line git (git rm <file>). I committed this change and pushed back to this branch.

**Comment by jmfrenzel** [comment]
> > The changes seem ok to me.
> > However following the instructions `localisation/README.md` I understand that `*.tsv` files are being created from "source" xml. As a consequencewe do not need `tsv` files in version control. I suggest to remove them from the PR
> 
> We can remove them at this stage. In the future we might need them again if we have to communicate with translators who cannot run scripts.

This is totally fine, as long as these files are removed before the PR is being merged.

**Pull request merged**